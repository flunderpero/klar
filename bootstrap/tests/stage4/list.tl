let my_list = list()
let len = 0
let i = 0
let s = ""

my_list.push("First")
my_list.push("Hello")
my_list.push("Third")

print("length()")
len = my_list.length()
print(len)

loop 
    if i == my_list.length() then
        break
    end
    s = my_list.at(i)
    print(s)
    i = i + 1
end
