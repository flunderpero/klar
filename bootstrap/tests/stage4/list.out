length()
3
First
Hello
Third
